packages/chart/src/hooks/useReduceData.ts

```diff
@@ -2,8 +2,22 @@ function useReduceData(config, data) {
   // for combo charts check if all  Data Series selected to Bar;
   const isBar = config?.series?.every(element => element?.type === 'Bar')
   // for combo charts check if all Data series selected Line or dashed-md/sm/lg.
-  const isAllLine = config?.series?.every(el => el.type === 'Line' || el.type=== 'dashed-sm'|| el.type=== 'dashed-md' || el.type=== 'dashed-lg');
-
+  const isAllLine = config?.series?.every(el => el.type === 'Line' || el.type === 'dashed-sm' || el.type === 'dashed-md' || el.type === 'dashed-lg')
+  const isNumber = value => {
```

Comment:
Can this be combined with the isNumber lifted to CdcChart? Can we add comments to clarify any regex expressions added. I understand we're returning a true/false value here, but adding clarifying remarks around regex statements would be beneficial so developers don't have to look them up in the future.

Reply:
moved to core/helper function

Reply:
I moved it to core/helpers and imported it here and into CdcChart

Reply:
and explained the regex in the helper func